src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -153,7 +155,7 @@ uiModules.get('apps/management')
       }
 
       function refreshData() {
-        return getData($scope.advancedFilter);
+        return getData(self.advancedFilter);
```

Comment:
I think it's a case of starting a refactor and then giving up because it was way beyond the scope of the PR. :(